repos/system_upgrade/el7toel8/actors/pythoninformuser/actor.py

```diff
@@ -0,0 +1,26 @@
+from leapp.actors import Actor
+from leapp.libraries.common import reporting
+from leapp.tags import ChecksPhaseTag
+from leapp.models import Report
+
+message = """
+Warning: In RHEL 8, there is no 'python' command. Python 3 (backward incompatible) is the primary Python version and Python 2 is available with limited support and limited set of packages.
```

Comment:
```suggestion
In RHEL 8, there is no 'python' command. Python 3 (backward incompatible) is the primary Python version and Python 2 is available with limited support and limited set of packages.
```

Reply:
Anyway, as we discussed, it's possible to setup `alternatives` so you should ideally generate remediation instruction. See the function `report_with_remediation` where you would put info that user would setup python2 or python3 to be ablle to use the `python` command. I think this ismessage you would provide to the customer.

Reply:
PR in my fork merged